posthog/models/integration.py

```diff
@@ -243,6 +245,23 @@ def oauth_config_for_kind(cls, kind: str) -> OauthConfig:
                 id_path="sub",
                 name_path="email",
             )
+        elif kind == "google-sheets":
+            if not settings.GOOGLE_SHEETS_APP_CLIENT_ID or not settings.GOOGLE_SHEETS_APP_CLIENT_SECRET:
```

Comment:
Still unsure if we want to generally have creds per client, given the only thing that changes is the scopes.

Can we at least make these credentials _not_ sheets specific so that if we add more integrations later we can just update the scopes, rather than having to handle more secrets.